test/plugin_functional/test_suites/data_plugin/index_patterns.ts

```diff
@@ -20,20 +20,11 @@ import expect from '@kbn/expect';
 import { PluginFunctionalProviderContext } from '../../services';
 import '../../plugins/core_provider_plugin/types';
 
-export default function ({ getService, getPageObjects }: PluginFunctionalProviderContext) {
+export default function ({ getService }: PluginFunctionalProviderContext) {
   const supertest = getService('supertest');
-  const esArchiver = getService('esArchiver');
-  const PageObjects = getPageObjects(['common', 'settings']);
 
   describe('index patterns', function () {
     let indexPatternId = '';
-    before(async () => {
-      await esArchiver.loadIfNeeded(
-        '../functional/fixtures/es_archiver/getting_started/shakespeare'
-      );
-      await PageObjects.common.navigateToApp('settings');
-      await PageObjects.settings.createIndexPattern('shakespeare', '');
-    });
```

Comment:
relocated to top-level `before` that loads before both index patterns and search